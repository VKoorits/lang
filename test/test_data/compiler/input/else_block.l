var a = 7
if a < 2:
	a = a * 2
else:
	a = a / 2
a = 7
