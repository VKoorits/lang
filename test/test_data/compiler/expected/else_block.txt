CONSTANTS:
	1: 7
	2: 2
CODE:
1:	NEW_VAR 1
2:	PUSH 1
3:	PUSH_CONST 1
4:	STORE
5:	PUSH 1
6:	PUSH_CONST 2
7:	LESS
8:	JUMP_IF_NOT 0x68
9:	NEW_VAR 0
10:	PUSH 1
11:	PUSH 1
12:	PUSH_CONST 2
13:	MULTIPLY
14:	STORE
15:	DELETE_VAR 0
16:	GOTO 0x95
17:	NEW_VAR 0
18:	PUSH 1
19:	PUSH 1
20:	PUSH_CONST 2
21:	DIVISION
22:	STORE
23:	DELETE_VAR 0
24:	PUSH 1
25:	PUSH_CONST 1
26:	STORE
27:	DELETE_VAR 1
28:	EXIT
